.wordpress-org/icon.svg

```diff
@@ -6,7 +6,7 @@
    version="1.1"
    id="svg181"
    sodipodi:docname="comment-svgrepo-com.svg"
-   inkscape:export-filename="../Code/subversion/yoast-comment-hacks/assets/icon.svg"
+   inkscape:export-filename="../Code/subversion/comment-hacks/assets/icon.svg"
```

Comment:
can just delete the `inkscape` lines I think

Reply:
Cleaned-up & minified :+1: